### DISCUSSION TITLE
Brevo SMTP Not working

### DISCUSSION THREAD

**Discussion by Arinji2** [pocketbase/pocketbase#3436] [Q&A] [answered]
Hii, so I was trying to setup the brevo smtp thing for pocketbase, However when i try to send a test email i get the following error

Failed to send the test email. Raw error: 
454 4.7.0 Invalid user.

![image](https://github.com/pocketbase/pocketbase/assets/87296817/70faa888-be6d-4c89-b958-c228ddcb7a66)
![image](https://github.com/pocketbase/pocketbase/assets/87296817/5c3c5f45-e453-4734-be80-6a79d33821b0)

Am i doing something wrong? I even added the user if that was the issue
![image](https://github.com/pocketbase/pocketbase/assets/87296817/3cc8e0bb-c792-49f2-858c-1c6484a2844e)

**Comment by ganigeorgiev** [comment, marked as answer]
I haven't used Brevo and I'm not sure what could be the reason for the error.

Based on the docs at https://help.brevo.com/hc/en-us/articles/209463245, did you submit a ticket to enable SMTP for your Brevo account?

Also I'm not sure if the information is outdated, but based on https://help.brevo.com/hc/en-us/articles/7924908994450#h_01GYW5T7TM88M9E6V80VN3QFR0, the SMTP host address seems to be `smtp-relay.sendinblue.com`.

If that's not the case, double check that there are no spaces/typos in the host, username and password fields.

**Comment by Arinji2** [comment]
Intresting, I didnt know you had to activate it. I used to use sendinblue before, stopped using it a while back.. lemme check and report back :D

Also thanks for the insane reply speed, love you mate :DDD

**Comment by Arinji2** [comment]
Alright, so Brevo just responded and said my SMTP server has been activated, 

They mentioned this
The SMTP port listed by default, Port 587, is to be used without a secure connection. If you would like to use a secure connection, (SSL or TLS), please use Port 465.

So with 465 i get a timed out error, and with 587 the user not found error, any idea?

**Reply by ganigeorgiev** [reply]
Are you testing locally or using a VPS provider like Hetzner, DigitalOcean, Linode, etc.?

If the latter note that some providers have restrictions for the outgoing mail ports and require contacting them (see https://github.com/pocketbase/pocketbase/discussions/1064).

And again, double check that you use the correct username/password combination (_based on the [Brevo docs](https://help.brevo.com/hc/en-us/articles/7924908994450#h_01GYW5T7TM88M9E6V80VN3QFR0) for the password you have to use the SMTP Key field value_).

**Reply by Arinji2** [reply]
I checked with them, they said no :/ 
Also just checked my username password, i tried the specific smtp key and also the master one, both have the timeout thing.
Port 465 ^^

**Reply by Arinji2** [reply]
https://github.com/pocketbase/pocketbase/discussions/537#discussioncomment-5756238
It seems as if being on the latest version fixes it, how do i check my version of pocketbase?
I am using this docker file: https://github.com/muchobien/pocketbase-docker

**Reply by ganigeorgiev** [reply]
You can find your PocketBase version in the bottom-right corner in the Admin UI but I don't think the error is PocketBase related.

Also, if you want to "force" encryption you have to change the "TLS encryption" field, but in most cases 587 + StartTLS should work just fine.

**Reply by Arinji2** [reply]
I can try with Mailtrap if you want? Or would you recommend something else to test it out with?

**Reply by Arinji2** [reply]
the smtp server seems to be working from this site
https://www.gmass.co/smtp-test
![image](https://github.com/pocketbase/pocketbase/assets/87296817/9d209061-96f7-4e25-bc15-c3dd7c190640)

**Reply by Arinji2** [reply]
OH WAIT, IM DUMB SORRYYY, there was a prespace in the username, i checked for any postspacas lol, not prespaces. 

Thanks for the help and really quick responses :DD

**Comment by devidrichh1** [comment]
Brevo SMTP not working properly for us lately. Switched to iDealSMTP, which has been reliable and fast. SMTPmart, SendinBlue, Mailgun, and Postmark also delivered great results for our email campaigns.

**Comment by petercarl542-ops** [comment]
Sorry to hear Brevo SMTP isn’t working for you. I'd recommend switching to something like SMTPmart or Mailgun; both are reliable and easy to set up.
